components/create-post-page/IsbnEntry.tsx

```diff
@@ -1,77 +1,61 @@
 import {
-  Box,
   Button,
-  HStack,
-  Icon,
+  FormControl,
+  FormLabel,
   Input,
   InputGroup,
   InputRightElement,
-  Text,
-  useToast,
   VStack,
 } from "@chakra-ui/react";
-import { useBookQuery } from "@lib/hooks/book";
-import { PopulatedBook } from "@lib/services/book";
 import { FormEvent, useState } from "react";
-import BarcodeIcon from "./BarcodeIcon";
 
 interface Props {
-  onIsbnSubmitted: (book: PopulatedBook) => void;
+  onIsbnSubmitted: (isbn: string) => void;
 }
 
 export default function IsbnEntry({ onIsbnSubmitted }: Props) {
   const [isbn, setIsbn] = useState("");
-  const toast = useToast();
-
-  const bookQuery = useBookQuery(isbn);
 
   const handleFormSubmit = (event: FormEvent) => {
-    if (isbn.length > 0 && bookQuery.data) {
-      onIsbnSubmitted(bookQuery.data);
-    } else {
-      toast({
-        title: "Book not found",
-        description:
-          "Unfortunately, that book doesn't appear to be required for any courses. " +
-          "Please double check that you've entered the ISBN correctly.",
-        status: "error",
-        duration: 5000,
-        isClosable: true,
-      });
-    }
+    onIsbnSubmitted(isbn);
     event.preventDefault();
   };
 
   return (
-    <Box backgroundColor="accent" rounded={10} p={2}>
-      <VStack>
-        <Icon as={BarcodeIcon} w={100} h={100} color="secondaryBackground" />
-        <HStack spacing={4}>
-          <form onSubmit={handleFormSubmit}>
-            <InputGroup>
-              <Input
-                backgroundColor="primaryBackground"
-                aria-label="isbn"
-                onChange={(e) => setIsbn(e.target.value.trim())}
-              />
-              <InputRightElement width="6.5rem">
-                <Button
-                  type="submit"
-                  size="sm"
-                  isLoading={bookQuery.isLoading}
-                  colorScheme="teal"
-                  isDisabled={isbn.length === 0}
-                >
-                  Find Book
-                </Button>
-              </InputRightElement>
-            </InputGroup>
-          </form>
-        </HStack>
-        <Text as="i" color="secondaryBackground">
-          Type ISBN
-        </Text>
-      </VStack>
-    </Box>
+    <VStack width="100%" maxWidth="xs">
+      <form onSubmit={handleFormSubmit} style={{ width: "100%" }}>
+        <FormControl>
+          <FormLabel
+            textAlign="center"
+            htmlFor="isbn"
+            color="secondaryText"
+            mb="3"
+          >
+            or select book by ISBN...
```

Comment:
Consider doing something like this instead:
![Screen Shot 2022-03-29 at 10 01 21 PM](https://user-images.githubusercontent.com/40670885/160738414-84111115-51a7-4527-98a2-8c36197c8541.png)

Reply:
I tried this first but didn't like it